test/e2e/theia_clickhouse_test.go

```diff
@@ -0,0 +1,382 @@
+	getInsertRateCmd = "./theia clickhouse status --insertRate"
+	insertQuery      = `INSERT INTO flows (
+                   flowStartSeconds,
+                   flowEndSeconds,
+                   flowEndSecondsFromSourceNode,
+                   flowEndSecondsFromDestinationNode,
+                   flowEndReason,
+                   sourceIP,
+                   destinationIP,
+                   sourceTransportPort,
+                   destinationTransportPort,
+                   protocolIdentifier,
+                   packetTotalCount,
+                   octetTotalCount,
+                   packetDeltaCount,
+                   octetDeltaCount,
+                   reversePacketTotalCount,
+                   reverseOctetTotalCount,
+                   reversePacketDeltaCount,
+                   reverseOctetDeltaCount,
+                   sourcePodName,
+                   sourcePodNamespace,
+                   sourceNodeName,
+                   destinationPodName,
+                   destinationPodNamespace,
+                   destinationNodeName,
+                   destinationClusterIP,
+                   destinationServicePort,
+                   destinationServicePortName,
+                   ingressNetworkPolicyName,
+                   ingressNetworkPolicyNamespace,
+                   ingressNetworkPolicyRuleName,
+                   ingressNetworkPolicyRuleAction,
+                   ingressNetworkPolicyType,
+                   egressNetworkPolicyName,
+                   egressNetworkPolicyNamespace,
+                   egressNetworkPolicyRuleName,
+                   egressNetworkPolicyRuleAction,
+                   egressNetworkPolicyType,
+                   tcpState,
+                   flowType,
+                   sourcePodLabels,
+                   destinationPodLabels,
+                   throughput,
+                   reverseThroughput,
+                   throughputFromSourceNode,
+                   throughputFromDestinationNode,
+                   reverseThroughputFromSourceNode,
+                   reverseThroughputFromDestinationNode)
+                   VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?)`
+	recordPerCommit      = 1000
+	insertInterval       = 1
+	threshold            = 25
+	MaxInt32             = 1<<31 - 1
+	numFieldsInDiskInfo  = 9
+	numFieldsInTableInfo = 7
+	dateBaseName         = "default"
+	defaultPath          = "/var/lib/clickhouse/"
+	// 8 GiB
+	chStorageSize = 8.0
+	sizeUnit      = "GiB"
+)
+
+var tableColumnNumberMap = map[string]string{
+	".inner.flows_node_view":   "16",
+	".inner.flows_pod_view":    "20",
+	".inner.flows_policy_view": "27",
+	"flows":                    "49",
+}
+
+func TestTheiaStatusCommand(t *testing.T) {
```

Comment:
Maybe also add ClickHouse to the test name?

Reply:
Added. Thanks.